pybossa/util.py

```diff
@@ -1195,4 +1195,15 @@ def description_from_long_description(desc, long_desc):
         text_desc += "..."
     description = blank_space_regex.sub(" ", text_desc)
     return description if description else " "
-    
\ No newline at end of file
+    
+def get_user_type(firm_num):
+    firm_to_type = current_app.config.get('FIRM_TO_TYPE', {})
+    if firm_num:
```

Comment:
If firm_num is None, we can make it return 'None'. could refactor:
` return firm_to_type.get(int(firm_num), None) if firm_num else None`